topic_monitor/setup.py

```diff
@@ -1,17 +1,27 @@
-from ament_python.data_files import get_data_files
-from ament_python.script_dir import install_scripts_to_libexec
+import sys
+
 from setuptools import find_packages
 from setuptools import setup
 
 package_name = 'topic_monitor'
-data_files = get_data_files(package_name)
-install_scripts_to_libexec(package_name)
+
+# install scripts into libexec path
+if 'develop' in sys.argv:
+    sys.argv[sys.argv.index('develop') + 1:sys.argv.index('develop') + 1] = \
```

Comment:
> ament tools isn't part of the debian package build. Can you explain why the PR to ament_tools would be needed to validate a debian build with this change?

I wasn't saying that. I just wanted to point out that you can take a closer look to how `ament_tools` builds Python packages. Since it works in that case the Debian rules file should be able to replicate the same invocations and achieve the same behavior.

> We added the install scripts together while I was in CA with ros2/bloom@31fee87 and its escape fixes ros2/bloom@f221ee7 because the setting of `--prefix` alone wasn't enough to put the non-libexec binaries in the ROS installation prefix.

Please try a local build with the modified `ament_tools` package. It doesn't pass the `--install-scripts` argument explicitly yet all executables are installed into the right location (at least for me). If you see a different behavior we need to debug *why* that is the case. But if that works for you too it should be possible to also achieve the same behavior from the Debian rule without (without the need to pass that argument).